admin/src/assets/styles/layouts/_Onboarding.scss

```diff
@@ -0,0 +1,272 @@
+					margin-bottom: 10px;
+				}
+			}
+		}
+	}
+
+
+	&-content {
+		margin: 0 20px 40px;
+
+		&-wrapper {
+			width: 100%;
+			margin: 0 auto;
+
+			&.small-wrapper {
+				max-width: 500px;
+			}
+
+			&.large-wrapper {
+				max-width: 920px;
+			}
+		}
+
+		&-heading {
+			color: $black;
+			text-align: center;
+			margin-bottom: 32px;
+
+			.heading-title {
+				font-size: 24px;
+				font-weight: $font-weight-semi;
+			}
+
+			.heading-description {
+				font-size: 13px;
+				font-weight: $font-weight-normal;
+			}
+		}
+
+		&-settings {
+			color: $black;
+			background-color: $white;
+			border-radius: $br-10;
+			filter: $box-shadow-small;
+			padding: 20px 30px;
+
+			&-footer {
+				margin-top: 20px;
+
+				.footer-text {
+					font-size: 13px;
+
+					a.urlslab-button.simple {
+						padding: 0;
+					}
+				}
+			}
+
+			.urlslab-main-settings {
+				margin-bottom: 16px;
+			}
+
+			.urlslab-advanced-settings-toggle {
+				margin-top: 20px;
+			}
+
+			.urlslab-advanced-settings {
+				margin-top: 20px;
+			}
+		}
+
+		// Customize styling for specific step
+		.step-modules {
+			.urlslab-onboarding-content-settings {
+				padding: 0;
+				overflow-y: auto;
+
+				&-footer {
+					margin: 0;
+					padding: 20px 30px;
+					position: sticky;
+					bottom: 0;
+					width: 100%;
+					background: $white;
+					border-top: 1px solid $grey-light;
+					z-index: 2;
+				}
+
+				&-modules {
+					@include scrollbarWithBg;
+					max-height: 525px;
+					overflow-y: auto;
+
+					.urlslab-dashboardmodule {
+						padding: 20px 30px 25px;
+						margin: 0;
+						width: 100%;
+						box-shadow: none;
+						filter: none;
+						border-radius: 0;
+
+						&:not(:last-child) {
+							border-bottom: 1px solid $grey-light;
+						}
+
+						&:hover {
+							box-shadow: none;
+						}
+
+						&-top {
+							border-bottom: 0;
+							margin: 0 0 20px;
+							padding: 0;
+						}
+
+						&-title[class] {
+							font-size: 16px;
+						}
+
+						&-content {
+							p {
+								font-size: 12px;
+							}
+						}
+
+						.urlslab-switch-text {
+							font-size: 14px;
+							line-height: 16px;
+						}
+					}
+				}
+			}
+
+
+		}
+
+		.urlslab-onboarding-success-wrapper {
+			font-size: 13px;
+			font-weight: $font-weight-semi;
+			color: $white;
+			background-color: $saturated-green;
+			padding: 5px 15px;
+			margin-bottom: 20px;
+			border-radius: 100px;
+
+			svg {
+				@include square(24px);
+
+				path {
+					fill: currentColor
+				}
+			}
+		}
+
+		.urlslab-half-columns {
+			display: flex;
+			flex-direction: row;
+			flex-wrap: wrap;
+			justify-content: space-between;
+			width: 100%;
+
+			&-col {
+				width: 50%;
+				margin-bottom: 16px;
+
+				&:first-child {
+					padding-right: 15px;
+				}
+
+				&:last-child {
+					padding-left: 15px;
+				}
+
+				&>* {
+					width: 100%;
+					max-width: 100%;
+				}
+			}
+
+			@media (max-width: $breakpoint-mobile) {
+				flex-direction: column;
+
+				&-col {
+					width: 100%;
```

Comment:
not needed

Reply:
this one should be there, otherwise it's 50% width from outside @media